src/VCR/RequestMatcher.php

```diff
@@ -1,102 +0,0 @@
-<?php
-
-namespace VCR;
-
-/**
- * Collection of matcher methods to match two requests.
- */
-class RequestMatcher
```

Comment:
I think the BC break is that the request matchers changed. They used to be `callable`s and now are objects. Doing a `addRequestMatcher` and using that with `enableRequestMatchers` in the current way is not possible http://php-vcr.github.io/documentation/configuration/.